extensions/parquet/table/src/main/java/io/deephaven/parquet/table/metadata/SortColumnInfo.java

```diff
@@ -0,0 +1,61 @@
+//
+// Copyright (c) 2016-2024 Deephaven Data Labs and Patent Pending
+//
+package io.deephaven.parquet.table.metadata;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import io.deephaven.annotations.SimpleStyle;
+import io.deephaven.api.ColumnName;
+import io.deephaven.api.SortColumn;
+import org.immutables.value.Value.Check;
+import org.immutables.value.Value.Immutable;
+import org.immutables.value.Value.Parameter;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Immutable
+@SimpleStyle
+@JsonSerialize(as = ImmutableSortColumnInfo.class)
+@JsonDeserialize(as = ImmutableSortColumnInfo.class)
```

Comment:
Would be good to have an explicit test for ser/deser; ditto wrt TableInfo.

Reply:
We have end-to-end tests, and I have tested with a legacy file.